test/e2e/storage/volume_provisioning.go

```diff
@@ -419,9 +418,7 @@ var _ = SIGDescribe("Dynamic Provisioning", func() {
 				defer deleteStorageClass(c, class.Name)
 
 				claim := newClaim(*betaTest, ns, "beta")
-				claim.Annotations = map[string]string{
```

Comment:
I think you should remove whole  `if betaTest != nil {` branch as there is no need to test the beta.

Reply:
I will leave the test until we decide when to graduate v1beta1 to v1 :)